### ISSUE TITLE
Why the name LitElement and not Polymer v4?

### ISSUE THREAD

**Issue by LukeTOBrien** [Polymer/lit-element#1006]
Hello folks!

I think this is the most important issue facing mankind at the moment.  
You are called "The Ploymer Project" and yet Ploymer itself is deprecated in favour of something called "LitElement".  
Are you now going to rename yourself "The LitElement Project"?  
I think the name "Polymer" is better than "LitElement" and you have a nice logo, LitElement doesn't have a logo.

I have been studying your guides and I do like what I see, I am planning on using LitElement on a new TypeScript project, but this one issue keeps going round in my head.

I would also like to point out that the footer on your [website](https://www.polymer-project.org/) says copyright 2018.  
To the untrained eye it could look like you haven't worked on it for two years, so I suggest a script to always set the copyright to the current year?

**Comment by arthurevans** [comment]
You are not wrong on either point. The branding is a little confusing, and the copyrights are out of date. We can fix one of those things in the short term...

As to why it's not Polymer 4... Well, LitElement builds on ideas developed while working on the Polymer library, but it doesn't really share any code with the Polymer library, so it would probably be more confusing to call it Polymer 4. Hope that helps.

**Comment by jaredcwhite** [comment]
I think the Polymer brand got muddled up a few years ago in its infancy…I remember it as "that weird experimental thing which generates components which only work with polyfills, and maybe a related design system…?"

Meanwhile, I remember when LitElement first came to my awareness last year, and I looked at it and thought it looked really cool, and then I noticed the fine print and thought "Oh, so _this_ is where Polymer eventually ended up. Groovy! This looks like something I'll actually want to use now!"

I could be totally off on all of this, but I'm just sharing my gut reactions over the years.

**Comment by LukeTOBrien** [comment]
Thanks for clearing that up!

If you are interested, I have already started creating my components and I have created [demos on CodePen](https://codepen.io/collection/DbaoQG?cursor=ZD0wJm89MSZwPTEmdj0yOTg0NTMw).  
It's still very early days, I need to improve a lot and there are more to come, I hope one day I will make it an open source project.

Most of the components are jQuery plugins that I made before TypeScript was invented and I've rewritten them in ~~Polymor~~ LitElement components.
